GraphicEditor/Resources/Styles/RectangleStyles.xaml

```diff
@@ -1,15 +1,24 @@
 ﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
                     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
 
-    <Style x:Key="DoorRectangle" TargetType="Rectangle">
+    <Style x:Key="DoorHorizontalRectangle" TargetType="Rectangle">
         <Setter Property="Fill" Value="#FF61CCF0" />
         <Setter Property="Stroke" Value="Black" />
         <Setter Property="StrokeThickness" Value="3" />
         <Setter Property="Height" Value="10" />
         <Setter Property="HorizontalAlignment" Value="Left" />
         <Setter Property="VerticalAlignment" Value="Top" />
         <Setter Property="Width" Value="50" />
-        <Setter Property="RenderTransformOrigin" Value="0.5,0.5" />
+    </Style>
+
+    <Style x:Key="DoorVerticalRectangle" TargetType="Rectangle">
+        <Setter Property="Fill" Value="#FF61CCF0" />
+        <Setter Property="Stroke" Value="Black" />
+        <Setter Property="StrokeThickness" Value="3" />
+        <Setter Property="Height" Value="50" />
+        <Setter Property="HorizontalAlignment" Value="Left" />
+        <Setter Property="VerticalAlignment" Value="Top" />
+        <Setter Property="Width" Value="10" />
     </Style>
 
```

Comment:
<Style x:Key="DoorVerticalRectangle" 
           BasedOn="DoorHorizontalRectangle" 
           TargetType="Rectangle">

You should do something like this :)